pkg/lockfile/dpkg-status.go

```diff
@@ -0,0 +1,105 @@
+package lockfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const DebianEcosystem Ecosystem = "Debian"
+
+func groupDpkgPackageLines(scanner *bufio.Scanner) [][]string {
+	var groups [][]string
+	var group []string
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if line == "" {
+			if len(group) > 0 {
+				groups = append(groups, group)
+			}
+			group = make([]string, 0)
+
+			continue
+		}
+		group = append(group, line)
+	}
+
+	if len(group) > 0 {
+		groups = append(groups, group)
+	}
+
+	return groups
+}
+
+func parseDpkgPackageGroup(group []string, pathToLockfile string) PackageDetails {
+	var pkg = PackageDetails{
+		Ecosystem: DebianEcosystem,
+		CompareAs: DebianEcosystem,
+	}
+
+	// TODO File SPECS:
+	for _, line := range group {
+		switch {
+		case strings.HasPrefix(line, "Package:"):
```

Comment:
Note that according to the OSV-Schema (https://ossf.github.io/osv-schema/#affectedpackage-field), the package name for the "Debian" ecosystem should be the **source** package name. This is also what the existing Debian DB on OSV keys on.

Reply:
Thanks for the info, I totally missed that!
So, I will refer to `Source: ` filed.
Analyzing it further, my understanding is that there are situations in which an override to the source package version is present in brackets when it differs from that of the binary package.
Example:
```
Package: bsdutils
Source: util-linux (2.36.1-8+deb11u1)
Version: 1:2.36.1-8+deb11u1
```
I will double check this and update PR,